scripts/global.js

```diff
@@ -292,6 +292,7 @@ export async function start() {
         domDisplaySettingsBtn: document.getElementById('settingsDisplayBtn'),
         domVersion: document.getElementById('version'),
         domFlipdown: document.getElementById('flipdown'),
+        domTesnetToggler: document.getElementById('tesnetToggler'),
```

Comment:
Typo: `domTesnetToggler` --> `domTestnetToggler`, the typo is consistent around the PR too.